README.md

```diff
@@ -1 +1,167 @@
-# weather-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>WeatherAPP </b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [WeatherAPP] <a name="about-project"></a>
+
+**WeatherAPP** is a project that allows users to see the weather of any city in the world. It is built with React and Redux.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React</li>
+    <li>Redux</li>
+    <li>Webpack</li>
+    <li>Jest</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Realtime weather info**
+- **Search for any city in the world**
+- **Responsive design**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Click [here](https://weather-app-ln0n.onrender.com/)
+
+## Presentation
+
+[Watch the video](https://drive.google.com/file/d/1XDVBzCsGMfe4i34RJmy5X1Z7LC6WFHNb/view?usp=sharing)

```

Comment:
- [ ] Please mention a piece of code you find interesting or you are proud of in your presentation video as mentioned in the project requirements. You only showed the demo and mentioned the features of the project.